src/angularfire2.ts

```diff
@@ -2,8 +2,9 @@ import * as firebase from 'firebase/app';
 import * as utils from './utils';
 import { FirebaseAppConfigToken, FirebaseApp, _firebaseAppFactory } from './app/index';
 import { FirebaseListFactoryOpts, FirebaseObjectFactoryOpts, FirebaseAppConfig } from './interfaces';
-import { FirebaseAppName } from './tokens';
-import { Injectable, OpaqueToken, NgModule } from '@angular/core';
+import { Injectable, InjectionToken, OpaqueToken, NgModule } from '@angular/core';
```

Comment:
It unfortunately can't. TypeScript throws a fit because `OpaqueToken` is used underneath the hood.